tests/xtro-sharpie/common-Network.ignore

```diff
@@ -0,0 +1,28 @@
+# the following protocols ae binded manualy as NativeObject
+
+!missing-protocol! OS_nw_advertise_descriptor not bound
+!missing-protocol! OS_nw_connection not bound
+!missing-protocol! OS_nw_content_context not bound
+!missing-protocol! OS_nw_endpoint not bound
+!missing-protocol! OS_nw_error not bound
+!missing-protocol! OS_nw_interface not bound
+!missing-protocol! OS_nw_listener not bound
+!missing-protocol! OS_nw_parameters not bound
+!missing-protocol! OS_nw_path not bound
+!missing-protocol! OS_nw_path_monitor not bound
+!missing-protocol! OS_nw_protocol_definition not bound
+!missing-protocol! OS_nw_protocol_metadata not bound
+!missing-protocol! OS_nw_protocol_options not bound
+!missing-protocol! OS_nw_protocol_stack not bound
+
+# binded but with a name alias changes and xtro gets confused.
+!missing-pinvoke! nw_ip_create_metadata is not bound
+!missing-pinvoke! nw_udp_create_metadata is not bound
+
+# binded via a Marshal.ReadIntPtr (Dlfcn.dlsym ..
+!missing-field! _nw_parameters_configure_protocol_default_configuration not bound
+!missing-field! _nw_parameters_configure_protocol_disable not bound
+!missing-field! _nw_content_context_default_message not bound
+!missing-field! _nw_content_context_default_stream not bound
+!missing-field! _nw_content_context_final_send not bound
+!missing-field! _nw_connection_send_idempotent_content not bound
```

Comment:
hmm... maybe we should move them to the binding file (with the enum) so we can have our existing tests run on them

Reply:
_nw_parameters_configure_protocol_default_configuration and _nw_parameters_configure_protocol_disable are BlockLiterals, the others are NWContentContext, does Field support that?

Reply:
no, but you can replace
```
... Dlfcn.dlsym (Libraries.Network.Handle, "_nw_content_context_default_message") ...
```
with
```
Network.Constants._nw_content_context_default_message
```
and have this defined in the binding files (under a static `Constants` type).